Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ruby:3.2.1-alpine3.17
+FROM ruby:3.2.4-alpine3.18
```

Comment:
Is there any reason we wouldn't want to move to ruby 3.3.1 and Alpine 3.19?

We have a PR raised against pact-broker-docker for the same

https://github.com/pact-foundation/pact-broker-docker/pull/185

Reply:
When I was doing the deps update for pactflow I checked for the latest ruby version and I saw 3.2.4 as the latest for some reason and that is why I updated the version on pact-broker to 3.2.4 as well. Coming to the alpine version I'll update it on pactflow and broker the next time I do the deps update.